content/en/blog/2023/kubecon-na.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon NA 2023
+linkTitle: KubeCon NA '23
+date: 2023-09-11
+# prettier-ignore
+cSpell:ignore: Anusha Aronoff Benedikt Bongartz Broadbridge Chronosphere Contribfest Coralogix Danielson Endo Flamegraphs Hrabovcak Itiel Itoh Jaglowski Kanal Komodor Kota Masanori Matej Mirabella Narapureddy observ Ohly Pivotto Purvi Reddy Sharone Shishi Shivanshu Shrivastava Shwartz Zitzman
+author: '[Severin Neumann](https://github.com/svrnm) (Cisco)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are thrilled to be at [KubeCon NA][] in Chicago from
+November 6 - 9, 2023.
+
+Read on to learn about all the things related OpenTelemetry during KubeCon.
+
+## OpenTelemetry Contribfest
+
+We are happy to announce, that for the first time, you can join the
+OpenTelemetry maintainers in making OpenTelemetry better for everyone during the
+[OpenTelemetry Contribfest](https://sched.co/1JWS7), on Wednesday November 8,
+2023 from 2:30pm - 3:55pm. There will be several opportunities to contribute:
+
+- Join the Collector maintainers and help fix bugs, add new features, and add
+  new transformation functions.
+- Help the OTel JS maintainers update the instrumentation libraries and ensure
+  they are outputting semantic convention-compliant telemetry data.
```

Comment:
Maybe want to link to https://opentelemetry.io/docs/specs/semconv/ for context?

Reply:
```suggestion
  they are outputting [semantic convention-compliant](https://opentelemetry.io/docs/specs/semconv/) telemetry data.
```